src/docs/community/migration.mdx

```diff
@@ -0,0 +1,1206 @@
+    module.exports =  {
+        ...
+        extends:  [ '@pxblue/eslint-config/ts' ],
+    };
+
+    // after
+    module.exports =  {
+        ...
+        extends:  [ '@brightlayer-ui/eslint-config/ts' ],
+    };
+    ```
+
+## @brightlayer-ui/icons
+
+-   Change package namespace from `@pxblue` to `@brightlayer-ui`.
+
+    ```sh
+    // package.json
+
+    @pxblue/icons => @brightlayer-ui/icons
+    ```
+
+-   Code snippet to use icon font in angular application.
+
+    ```
+    // angular.json
+    // before
+    "styles": [
+        "src/styles.scss",
+        "./node_modules/@pxblue/icons/iconfont/PxblueUIIcons.css"
+    ],
+
+    // after
+    "styles": [
+        "src/styles.scss",
+        "./node_modules/@brightlayer-ui/icons/iconfont/BrightlayerUIIcons.css"
+    ],
+    ```
+
+-   Code snippet to use icon font in React application.
+
+    ```
+    // index.js
+
+    // before
+    require('@pxblue/icons/iconfont/PxblueUIIcons.css');
+
+    // after
+    require('@brightlayer-ui/icons/iconfont/BrightlayerUIIcons.css');
+    ```
+
+-   Change all `pxb-` class name prefixes to `blui-` in component
+
+    ```
+    // before
+    <i class="pxb-ICONNAME"></i>
+
+    // after
+    <i class="blui-ICONNAME"></i>
+    ```
+
+## @brightlayer-ui/icons-mui
+
+-   Change package namespace from `@pxblue` to `@brightlayer-ui`.
+
+    ```sh
+    // package.json
+
+    @pxblue/icons-mui => @brightlayer-ui/icons-mui
+    ```
+
+-   Replace package name from `@pxblue` to `@brightlayer-ui` to use mui icon in React application.
+
+    ```
+    \\ before
+    import myIcon from '@pxblue/icons-mui/MyIcon';
+
+    \\ after
+    import myIcon from '@brightlayer-ui/icons-mui/MyIcon';
+
+    ```
+
+## @brightlayer-ui/icons-svg
+
+-   Change package namespace from `@pxblue` to `@brightlayer-ui`.
+
+    ```sh
+    // package.json
+
+    @pxblue/icons-svg => @brightlayer-ui/icons-svg
+    ```
+
+-   Code snippet to use svg icon in Angular application.
+
+    ```
+    // app.module.ts
+    import {MatIconModule} from '@angular/material/icon';
+    import {HttpClientModule} from '@angular/common/http';
+    ...
+    imports:[
+      MatIconModule,
+      HttpClientModule
+    ]
+    ```
+
+    The simplest way to use these SVGs in Angular is to register them with the matIconRegistry so they can be used with the `<mat-icon>` tag. You can register icons individually, or as the entire Brightlayer UI set:
+
+    ```
+    import { Component } from "@angular/core";
+    import { MatIconRegistry } from "@angular/material/icon";
+    import { DomSanitizer } from "@angular/platform-browser";
+
+    declare var require: any;
+    const icon = require("@brightlayer-ui/icons-svg/breaker.svg").default; // individual icon
+    const iconSet = require("@brightlayer-ui/icons-svg/icons.svg").default; // full set
+    ```
+
+    Then, in your constructor, register the icon or the icon set. It will then be available for use as a `<mat-icon>`.
+
+    #### Individual Icon
```

Comment:
If your intention is to add a subheading, then it won't work if you put four spaces in front of it.

Reply:
With above changes it is showing like below

<img width="768" alt="Screenshot 2021-12-17 at 8 24 46 PM" src="https://user-images.githubusercontent.com/10433274/146563227-19b0fe7a-9103-42cf-b9bb-381b1afc9cf0.png">

Reply:
Looks good to me... unless you intended to format it in a different way.

Reply:
I see... my bad.